src/js/player.js

```diff
@@ -1240,6 +1413,8 @@ class Player extends Component {
       // Don't trigger a change event unless it actually changed
       if (this.usingNativeControls_ !== bool) {
```

Comment:
I think I'd actually like to remove this ability to switch between native and custom controls mid-playback. The decision to use native controls or not should be made when the tech is first initialized, and then not changed. That would simplify the controls situation a bunch.

The getter part of `player.usingNativeControls` is still needed though. It would just return the value from the tech.